backend/src/nodes/properties/inputs/pytorch_inputs.py

```diff
@@ -20,9 +29,46 @@ def __init__(
     ):
         super().__init__(input_type, label)
 
-    def enforce(self, value):
+    def enforce(self, value: PyTorchModel):
+        if torch is not None:
+            assert isinstance(value, torch.nn.Module), "Expected a PyTorch model."
+            assert isPyTorchModel(value), "Expected a supported PyTorch model."
```

Comment:
Again, this doesn't make sense from a type perspective. You declared that `value` is a `PyTorchModel`, so all these assertions are supposed to always be true. Of course, they aren't because the type of `value` is wrong. We don't know the type of `value`, that's why we have these checks.

Please remove the type annotation on `value` in all `enforce` methods.